sphinxext/icon.py

```diff
@@ -47,6 +47,7 @@
         'eye',
         'globe',
         'lock',
+        'money',
```

Comment:
I believe `money` is a FontAwesome 4 name, and it's named differently in FA 5, so this would have to be down in ICONS_FA4